lib/yardstick/measurement.rb

```diff
@@ -34,7 +34,7 @@ def initialize(rule)
     #
     # @api public
     def ok?
-      @result == true || skip?
+      @result.equal?(true) || skip?
```

Comment:
I wounder why we even have to compare `@result` with `true`? Or can it have any other valid truthy value?